### ISSUE TITLE
Need GUI to select time-zone

### ISSUE THREAD

**Issue by GeoffWyche** [TC2013/Roundtrip2#8]

**Comment by timomer** [comment]
Would you not read this from the phone? Or does this need to be independent of the phone timezone?

**Comment by GeoffWyche** [comment]
Good point.  The time zone is needed because Medtronic pumps do not use
time zones in their timestamps.  The phone's time zone should be just
fine.

--G

**Comment by timomer** [comment]
Right, got you.

The only issue I can see here is when a user travels across time zones, do these users update the time on their pumps? The phone will most likely update itself, this will happen without the user noticing so maybe it is best to manually set in the app.

@TC2013 what are your thoughts on this?

**Comment by TC2013** [comment]
We haven't traveled using OpenAPS recently, but we used to have to set our phone not to switch time zones when traveling.  I'll check with @scottleibrand because I think they actually have a better workaround.  However, the pi/edison wouldn't change time, but getting BG values from NS might be affected (if the user is uploading CGM with their phone).

**Comment by scottleibrand** [comment]
Our loop runs clockset.sh to set the pump and CGM time to the current time on the rig if it is mismatched. If the rig doesn't have NTP sync'd, it instead sets the rig to the more recent of the pump and CGM times. A similar approach should work for you.

**Comment by timomer** [comment]
Cool, sounds good, so the rig will be our mobile. CGM time should be
correct as that's an android app, we will just need to be sure the pumps
time matches phone time.
